src/billing/List/BillingList.ts

```diff
@@ -0,0 +1,310 @@
+import _ from 'lodash/fp';
+import * as qs from 'qs';
+import { useEffect, useRef, useState } from 'react';
+import { div, h, h2, p, span } from 'react-hyperscript-helpers';
+import * as Auth from 'src/auth/auth';
+import { CreateBillingProjectControl } from 'src/billing/List/CreateBillingProjectControl';
+import { GCPNewBillingProjectModal } from 'src/billing/List/GCPNewBillingProjectModal';
+import { ProjectListItem, ProjectListItemProps } from 'src/billing/List/ProjectListItem';
+import { AzureBillingProjectWizard } from 'src/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard';
+import { GCPBillingProjectWizard } from 'src/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard';
+import ProjectDetail from 'src/billing/Project';
+import { isCreating, isDeleting } from 'src/billing/utils';
+import { billingRoles } from 'src/billing/utils';
+import { BillingProject, GoogleBillingAccount } from 'src/billing-core/models';
+import Collapse from 'src/components/Collapse';
+import { customSpinnerOverlay } from 'src/components/common';
+import { Ajax } from 'src/libs/ajax';
+import colors from 'src/libs/colors';
+import { reportErrorAndRethrow } from 'src/libs/error';
+import Events from 'src/libs/events';
+import * as Nav from 'src/libs/nav';
+import { useCancellation, useOnMount } from 'src/libs/react-utils';
+import * as StateHistory from 'src/libs/state-history';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+import { useWorkspaces } from 'src/workspaces/common/state/useWorkspaces';
+import { CloudProvider, cloudProviderTypes } from 'src/workspaces/utils';
+
+const BillingProjectSubheader = ({ title, children }) =>
```

Comment:
Code came from https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/billing/List/List.ts, except that `TopBar`, `FooterWrapper`, and `navPaths` were removed.

I tried to convince git to recognize this as a move by doing a series of move/rename/edit commits. However, it still thinks it is a new file, probably because the whitespace changes resulted in a high number of changed lines. You can see the actual diff (hiding whitespace changes) [here](https://github.com/DataBiosphere/terra-ui/pull/4632/commits/0ce2117d771a9a88717f862d5745d62d6eb911a9?diff=unified&w=1).